Seer product updates (#13953)
getsentry/sentry-docs: docs/product/ai-in-sentry/seer/index.mdx

---
title: Seer
description: "Learn about Seer, Sentry's AI agent, and managing generative AI features."
sidebar_order: 21
---

> Seer is Sentry's AI agent that powers a set of features built on generative AI and ML that uses Sentry's rich context (issue details, tracing data, and profiles) to help you understand and resolve issues faster. 

Reviewer: ```suggestion
Seer is Sentry's AI debugging agent that uses Sentry's rich context (issue details, tracing data, logs, and profiles) to help you troubleshoot and fix errors and performance issues faster.
```

## Seer Capabilities

Seer provides two main capabilities to enhance your debugging:

> - [**Issue Fix**](#issue-fix): A collaborative workflow to find the root cause and solution of issues
> - [**Issue Scan**](#issue-summary): Intelligently analyzes your issues and provides actionability scores to help you prioritize what to work on.

Reviewer: ```suggestion
- [**Issue Scan**](#issue-summary): Scans issues as they are ingested in Sentry to determine an actionability scoring and run Seer's issue fix on them automatically.
- [**Issue Fix**](#issue-fix): Automated root cause analysis and suggested code fixes.
```

### Issue Fix

Issue Fix uses AI to find the root cause of your issues and suggest solutions. WIth Issue Fix, you can:

- Get automated root cause analysis using Sentry's context and your codebase
- Receive AI-generated code fixes and unit tests
- Collaborate with Seer throughout the debugging process
- Create pull requests directly from the Seer Interface

> - Benefit from Github integration for seamless code changes

Reviewer: ```suggestion
- Benefit from GitHub integration for seamless code changes
```

Learn more about [Issue Fix](./issue-fix/).

### Issue Scan

> Issue Scan intelligently analyzes your issues and helps you prioritize what to work on. With Issue Scan, you can:

Reviewer: ```suggestion
Issue Scan intelligently analyzes your issues and helps you prioritize what to work on, and enables running Issue Fix automatically on an issue. With Issue Scan, you can:
```

- Filter issues by their latest scan results
- View actionability scores that indicate how likely an issue is to be resolvable 
- Se Seer labels in the issue stream for quick identification

> - Get insights into which issues need immediate attention

Reviewer: ```suggestion
- Get insights into which issues need immediate attention
- Enable Seer's Issue Fix to be started automatically once an issue is ingested in Sentry
```

## How Seer Works

> Seer leverages multiple data sources to provide accurate analysis and solutions:

Reviewer: Also logs

- **Issue Context**: Error messages, stack traces, and event metadata from your [Issues]()
- **Tracing Data**: Distributed [traces]() and span information

> - **Your Codebase**: Relevant code from linked GitHub repositories

Reviewer: ```suggestion
- **Your Codebases**: Relevant code from linked GitHub repositories
```

- **Performance Data**: Profiles and performance metrics
- **Interactive Feedback**: Your input and guidance during the process

## Getting Started with Seer

To start using Seer's capabilities:

1. **Enable Seer**: Make sure AI features are enabled in your [organization settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/#hideAiFeatures)
2. **Set up GitHub Integration**: For Issue Fix capabilities, [connect your GitHub repositories](/organization/integrations/source-code-mgmt/github/)
3. **Configure Settings**: Customize Seer's behavior in your project settings
4. **Start Using**: Look for Seer options in your issue details pages

> ## Data Processing for Generative AI

Reviewer: Should this go in the privacy page instead_

Reviewer: Up to you, as long as the info is consistent
